data/vtlib/pre_alignment.json

```diff
@@ -39,7 +39,12 @@
 		"type":"EXEC",
 		"use_STDIN": true,
 		"use_STDOUT": true,
-		"cmd":[ "bamadapterclip", "verbose=0", "level=0" ],
+		"cmd":{"select":"realignment_switch", "select_range":[1], "default":0, "comment":"detect and clip for realign, otherwise just clip (realignment is non-default)",
```

Comment:
```suggestion
		"cmd":{"select":"realignment_switch", "select_range":[1], "default":0, "comment":"Detect and clip for realign, otherwise just clip (realignment is non-default)",
```